components/atoms/Accordion.js

```diff
@@ -13,7 +13,7 @@ export function Accordion(props) {
   const arrowColor = "#D3080C";
 
   return (
-    <div id="accordion" className="flex mb-2">
+    <div id="accordion" className="flex mb-2 container">
```

Comment:
not sure about container but we can review later

Reply:
This was just so the accordion wasn't full bleed. Originally it was a child of the Main component which was just a ```<main>``` tag with a container slapped on it but since I removed that component, I added containers where necessary for now to keep the flow the same. Can definitely review later though.